lib/utils/request/src/index.js

```diff
@@ -42,15 +52,15 @@ const route = (template, parms) => {
   });
 };
 
+// Changes the request defaults abacus uses
+// Will override the same options or add the new one
+const setDefaults = (opts) => {
+  defopts = extend(defopts, opts);
+  return xrequest;
```

Comment:
AIUI the behavior you've coded here would be very different from the original behavior of request.defaults (which returns an instance of request pre-configured with the given defaults)